src/components/hr/hr.component.js

```diff
@@ -1,34 +1,24 @@
 import React from 'react';
-import PropTypes from 'prop-types';
+import propTypes from '@styled-system/prop-types';
 
 import StyledHr from './hr.style';
 
 const Hr = ({
-  mt = 3,
-  mb = 3,
-  ml,
-  mr
+  ...props
 }) => {
   return (
     <StyledHr
       data-component='hr'
-      mt={ mt }
-      mb={ mb }
-      ml={ ml }
-      mr={ mr }
+      mt={ props.mt || 3 }
```

Comment:
Would't this be better as a default prop that way it would be included in the prop table automatically?

Reply:
That would mean it would be included in both the component prop table and the spacing prop table in the docs which I didn't think was good